### PR TITLE
feat: add platformAppBridge

### PR DESCRIPTION
**Pull request by julianiff**
Hey,
i would like to add the platformAppBridge to use the Settings for Platform Apps.

### CODE REVIEW

File: packages/app-bridge/src/AppBridgePlatformApp.ts [hunk 1]
```diff
@@ -230,6 +230,18 @@ export class AppBridgePlatformApp implements IAppBridgePlatformApp {
...
+        return null as any;
+    }
+
+    getDocumentPagesByDocumentId(documentId: number): Promise<DocumentPage[]> {
+        return null as any;
+    }
+
+    getDocumentSectionsByDocumentPageId(documentPageId: number): Promise<DocumentSection[]> {
+        return null as any;
+    }
```
Comment (julianiff) [inline packages/app-bridge/src/AppBridgePlatformApp.ts:244]: Hey @SamuelAlev 
We have some typing issues and to fix some of them i have to add these methods. What do you think would be the best way to adapt these so we could use it for Platform Apps?

Comment (ragi96) [reply packages/app-bridge/src/AppBridgePlatformApp.ts:244]: This shouldn't be added in here, refactor the linkChooser so it displays something like `not supported for plattform apps` or just remove the internal linkSelector button if used with Plattform Apps

Comment (julianiff) [reply packages/app-bridge/src/AppBridgePlatformApp.ts:244]: What does it do?

Comment (ragi96) [reply packages/app-bridge/src/AppBridgePlatformApp.ts:244]: It's basically the linkChooser of a portal for internal links:
Screenshot 2024-02-08 at 13 30 44
And then opens this linkTree:
Screenshot 2024-02-08 at 13 31 15

**Comment by SamuelAlev** [comment]
Fixed in [Frontify/web-app@`eacb0d4` (#2830)](https://github.com/Frontify/web-app/pull/2830/commits/eacb0d4ca81ca0be514a482c25b7e4c5151fac6f)

**Pull request closed without merging**